### DISCUSSION TITLE
Installation issue with npm

### DISCUSSION THREAD

**Discussion by afif-malghani** [Microsoft/node-pty#625] [Q&A]
## Environment details

- OS: Ubuntu
- OS version: 22.04
- node-pty version: latest

## Issue description

installation fails with  at command  `sh -c node-gyp rebuild`

**Comment by Tyriar** [comment]
what happens?

**Comment by HilalNazli** [comment]
I was having the same issue. Re-installing Node.js like below solved the problem:
```
curl -fsSL https://deb.nodesource.com/setup_18.x | sudo -E bash - &&\
sudo apt-get install -y nodejs
```
